src/org.xtuml.bp.core/color/ooaofooa_package_spec.clr

```diff
@@ -34,89 +34,19 @@
   .// The following markings are derived from the instances in io.mdl/sql/file_io.pei.sql
   .//  where ExportOrdering.componentRoot == true (except for S_SYS, which is handled above)
   .//
-  .invoke markComponentRoot("S_DOM")
```

Comment:
Are the changes here related to relocatables?  Or just old specialized package stuff that you saw and removed.  I think this should be written about in the note.